km_api/know_me/models.py

```diff
@@ -500,6 +500,26 @@ class Meta(object):
         verbose_name = _('profile item list content')
         verbose_name_plural = _('profile item list content')
 
+    def get_entry_list_url(self, request=None):
```

Comment:
It is not really clear to me what this means. I would prefer `get_list_entry_list_url()` even though it is slightly unwieldy. What do you think?